recipes/vadr/meta.yaml

```diff
@@ -16,7 +16,10 @@ source:
     folder: fasta
 
 build:
-  number: 2
+  number: 0
+  skip: True  # [osx]
```

Comment:
What is the reason to disable the build for OSX ?

Reply:
The OSX build seems to have started failing in the previous version 1.6.2  (#44370). I don't have the necessary knowledge or resources to figure out why the OSX build is failing. It would be great if at least the Linux version of VADR were updated in Bioconda for those that would want to integrate the tool into their pipelines.

Reply:
I've rebased #44370 to see what's the produced error.

Reply:
I think the problem is in:
```
cp: .//testfiles/github-issues/iss70-cdsstopn3p/.#1.ntlist: No such file or directory[0m
2024-06-06T14:49:42.0167540Z 14:49:42 [32mBIOCONDA INFO[0m (ERR) Traceback (most recent call last):[0m
2024-06-06T14:49:42.0167970Z 14:49:42 [32mBIOCONDA INFO[0m (ERR)   File "/opt/mambaforge/envs/bioconda/bin/conda-build", line 11, in <module>[0m
2024-06-06T14:49:42.0168380Z 14:49:42 [32mBIOCONDA INFO[0m (ERR)     sys.exit(execute())[0m
2024-06-06T14:49:42.0168870Z 14:49:42 [32mBIOCONDA INFO[0m (ERR)   File "/opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/conda_build/cli/main_build.py", line 590, in execute[0m
2024-06-06T14:49:42.0169300Z 14:49:42 [32mBIOCONDA INFO[0m (ERR)     api.build([0m
2024-06-06T14:49:42.0169760Z 14:49:42 [32mBIOCONDA INFO[0m (ERR)   File "/opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/conda_build/api.py", line 250, in build[0m
2024-06-06T14:49:42.0170190Z 14:49:42 [32mBIOCONDA INFO[0m (ERR)     return build_tree([0m
2024-06-06T14:49:42.0171560Z 14:49:42 [32mBIOCONDA INFO[0m (OUT) x86_64-apple-darwin13.4.0-clang -D_FORTIFY_SOURCE=2 -isystem $PREFIX/include -mmacosx-version-min=10.9 -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fstack-protector-strong -O2 -pipe -fPIE -isystem $PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/vadr-1.6.2 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix  -DSHOW_HELP -DSHOWSIM -DUNIX -DTIMES -DHZ=100 -DMAX_WORKERS=4 -DTHR_EXIT=pthread_exit  -DM10_CONS -DFASTA_HOST='"your.fasta.host.here"' -DIS_LITTLE_ENDIAN -DUSE_MMAP -DUSE_FSEEKO -DHAS_INTTYPES -DSAMP_STATS -DPGM_DOC -DBIG_LIB64 -DLIB_MEM=12 -c -DFASTM -DTFAST  initfa.c -o init_tfm.o[0m
2024-06-06T14:49:42.0172930Z 14:49:42 [32mBIOCONDA INFO[0m (ERR)   File "/opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/conda_build/build.py", line 3638, in build_tree[0m
2024-06-06T14:49:42.0173380Z 14:49:42 [32mBIOCONDA INFO[0m (ERR)     packages_from_this = build([0m
2024-06-06T14:49:42.0173870Z 14:49:42 [32mBIOCONDA INFO[0m (ERR)   File "/opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/conda_build/build.py", line 2506, in build[0m
2024-06-06T14:49:42.0174300Z 14:49:42 [32mBIOCONDA INFO[0m (ERR)     utils.check_call_env([0m
2024-06-06T14:49:42.0174790Z 14:49:42 [32mBIOCONDA INFO[0m (ERR)   File "/opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/conda_build/utils.py", line 405, in check_call_env[0m
```

On Linux x86_64 it just [gives](https://dev.azure.com/bioconda/25074ded-3133-43ca-af80-895ab87b53cb/_apis/build/builds/60726/logs/26):
```
2024-06-06T14:30:11.9850788Z 14:29:47 [32mBIOCONDA INFO[0m (OUT) bin/vadr-run.sh (text): Patching[0m
2024-06-06T14:30:11.9851791Z 14:29:47 [32mBIOCONDA INFO[0m (OUT) etc/conda/activate.d/vadr.sh (text): Patching[0m
2024-06-06T14:30:11.9852395Z 14:29:47 [32mBIOCONDA INFO[0m (OUT) share/vadr-1.6.2/vadr/build_env_setup.sh (text): Patching[0m
2024-06-06T14:30:11.9853607Z 14:29:47 [32mBIOCONDA INFO[0m (OUT) /opt/conda/lib/python3.10/site-packages/conda_build/build.py:1523: UserWarning: file /opt/conda/conda-bld/vadr_1717683961148/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/share/vadr-1.6.2/vadr/testfiles/github-issues/iss70-cdsstopn3p/.#1.ntlist is a symlink with no target[0m
2024-06-06T14:30:11.9854504Z 14:29:47 [32mBIOCONDA INFO[0m (OUT)   warnings.warn(f"file {path} is a symlink with no target", UserWarning)[0m
```


https://github.com/ncbi/vadr/blob/master/testfiles/github-issues/iss70-cdsstopn3p/.%231.ntlist says that it is a symbolic link.

Locally (Linux x86_64) it gives me:
```
testfiles/github-issues/iss70-cdsstopn3p on  master 
❯ la
lrwxrwxrwx martin martin  57 B  Fri Jun  7 10:43:00 2024  .#1.ntlist ⇒ nawrocke@cbbdev12.be-md.ncbi.nlm.nih.gov.43990:1681991543
drwxrwxr-x martin martin 4.0 KB Fri Jun  7 10:43:00 2024  expected-files
.rw-rw-r-- martin martin 1.6 KB Fri Jun  7 10:43:00 2024  iss70-cdsstopn3p.testin
.rw-rw-r-- martin martin 336 B  Fri Jun  7 10:43:00 2024  iss70.fa

testfiles/github-issues/iss70-cdsstopn3p on  master 
❯ cat .#1.ntlist 
[bat error]: '.#1.ntlist': No such file or directory (os error 2)

testfiles/github-issues/iss70-cdsstopn3p on  master 
❯ file .#1.ntlist 
.#1.ntlist: broken symbolic link to nawrocke@cbbdev12.be-md.ncbi.nlm.nih.gov.43990:1681991543
```

Reply:
Sorry to be slow, @peterk87 and @martin-g . I've addressed this in a new release of vadr (1.6.4) that does not include the rogue `.#1.ntlist` file.

Reply:
Thanks for the update, @nawrockie !
I will update the PR to 1.6.4!